### DISCUSSION TITLE
Kaç Puanınız var ???

### DISCUSSION THREAD

**Discussion by ghost** [IBAX-io/go-ibax#973] [Soru Cevap - Türkçe]
Benim 25 k normalmi ??

**Comment by Skips1** [comment]
25 k kanka Ortalama olarak herkezde bu var korkma .d

**Reply by ghost** [reply]
:) Tamam sağol ne korkucamo benden korksun...

**Comment by huseyin03** [comment]
bendede öyle hatta tam olarak 25,134 puan var
